Dockerfile

```diff
@@ -1,12 +1,13 @@
-FROM postgres:9.6
+FROM postgres:10.1
 
-ENV PATH ~/.cargo/bin/:$PATH
 ENV CARGO_HOME /cargo
+ENV PATH $CARGO_HOME/bin:$PATH
 ENV SRC_PATH /src
 
 RUN apt-get update \
   && apt-get install -y --no-install-recommends \
-    ca-certificates curl git make gcc gcc-multilib postgresql-server-dev-$PG_MAJOR=$PG_VERSION python-pip \
+    ca-certificates curl git make gcc gcc-multilib postgresql-server-dev-$PG_MAJOR=$PG_VERSION \
```

Comment:
what does python-setuptools get us?

Reply:
Again, it wasn't working without it. I'll try removing it and I'll add a comment with the error.

Reply:
```
Collecting pgxnclient
  Downloading pgxnclient-1.2.1.tar.gz (87kB)
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
    ImportError: No module named setuptools
```